capabilities_matrix/_topics/cloud_providers.md

```diff
@@ -2,60 +2,60 @@
-| Floating IP Addresses                      | Yes                       | Yes             | Yes                         | Yes                        | No                                    |
-| Network Ports                              | Yes                       | Yes             | Yes                         | Yes                        | No                                    |
-| Load Balancer Inventory                    | Yes                       | Yes             | Yes                         | No                         | No                                    |
-| Create/Update/Delete Network               | No                        | No              | No                          | Yes                        | No                                    |
-| Create/Update/Delete Subnet                | No                        | No              | No                          | Yes                        | No                                    |
-| Create/Update/Delete Routers               | Yes                       | Yes             | No                          | Yes                        | No                                    |
-| Tag Mapping from Provider                  | Yes                       | Yes             | No                          | Yes                        | Yes                                   |
-| Tag Mapping to Provider                    | No                        | No              | No                          | No                         | No                                    |
+| Feature                                    | Amazon Web Services (AWS) | Microsoft Azure | Google Cloud Platform (GCP) | Red Hat OpenStack Platform | IBM PowerVC via the OpenStack API     | IBM Power Systems Virtual Servers |
+| ------------------------------------------ | ------------------------- | --------------- | --------------------------- | -------------------------- | ------------------------------------- | --------------------------------- |
+| Relationship Discovery                     | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | Yes                               |
+| Instance Drift Comparison                  | Yes                       | Yes             | No                          | Yes                        | No                                    | No                                |
+| Track Instance Genealogy                   | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | No                                |
+| Capacity & Utilization                     | Yes                       | Yes             | Yes                         | Yes                        | No                                    | No                                |
```

Comment:
Are we sure this is no?  We don't have utilization data, but we have consumed data I believe for CPU, memory and disk?

Reply:
I think there's a distinction between data collected through the inventory refresher and C&U collector. The C&U data is stored in special 'metrics' tables in the DB.

So I think your questions is: Can some of the C&U reporting be run using data from the 'vms' table? I don't know.

Reply:
That's correct @jaywcarman , Reporting/chargeback can use consumed resources collected from ems_refresh but that is not the same as c&u/metrics